pandas/tests/frame/methods/test_append.py

```diff
@@ -7,6 +7,18 @@
 
 
 class TestDataFrameAppend:
+    @pytest.mark.parametrize("klass", [Series, DataFrame])
+    def test_append(self, multiindex_dataframe_random_data, klass):
```

Comment:
maybe rename to test_append_multiindex (or test_multiindex is fine as not redundant)

Reply:
Updated + green